cucumber-jparest/src/main/java/uk/gov/homeoffice/digital/sas/cucumberjparest/ServiceRegistry.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.homeoffice.digital.sas.cucumberjparest;
+
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.stereotype.Component;
+
+/**
+* Registry for services and their location.
+ */
+@Getter
+@Setter
+@Component
+@AllArgsConstructor
+public class ServiceRegistry {
+
+  public static final String SERVICE_REGISTRY_SYSTEM_PROPERTY_NAME
+      = "cucumber.jparest.serviceRegistry";
+  private Map<String, String> services;
```

Comment:
I think this can be final too.

Reply:
It can't be: it is set in `StepDefinitionsTest`

Reply:
ah yes ofc, I didn't look carefully enough at this. Resolved.